examples/graphql-react-apollo/README.md

```diff
@@ -44,3 +44,5 @@ $ yarn test:e2e
 - [`src/mocks.js`](src/mocks.js) describes GraphQL operations to mock.
 - [`src/index.js`](src/index.js) conditionally enables mocking in `development` environment.
 - [`src/setupTests.js`](src/setupTests.js) enables mocking for unit tests via `beforeAll`/`afterAll` hooks.
+- [`public/mockServiceWorker.js`](public/mockServiceWorker.js) the Service Worker, created by running `node_modules/.bin/msw init public`.
```

Comment:
The docs use `npx msw init` command, maybe we should mention it here for consistency as well?
Do you see any issues it may have over running the local executable?

Reply:
No, I've just copied from your reply in one of the issues ;-)

Reply:
Done!